LICENSE-Paulov-t.md

```diff
@@ -0,0 +1,26 @@
+# Paulov-t UNLICENSED
+
+Copyright (c) 2021-2024 [Paulov-t](https://github.com/paulov-t). All rights reserved.
+
+THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+**Paulov-t work is unlicensed**
+
+Any work or fork that is not Stay in Tarkov and uses any line of code from [Paulov-t](https://github.com/paulov-t), including but not limited to previous commits and forks of old commits is strictly prohibited. If you are using any [Paulov-t](https://github.com/paulov-t) code in your works, whether currently or previously, you can request permission directly from [Paulov-t](https://github.com/paulov-t) via GitHub, Email or Discord.
+
+**Redistribution**
+- You cannot reproduce and distribute copies of the Work (by [Paulov-t](https://github.com/paulov-t)) or Derivative Works (by [Paulov-t](https://github.com/paulov-t)) thereof in any medium, with or without modifications, and in Source or Object form.
+
+**Expiry of old NCSA Open Source License and this replacement**: 
+-  After the project was moved out of [Paulov-t](https://github.com/paulov-t) to [Stay In Tarkov](https://github.com/stayintarkov). [Paulov-t](https://github.com/paulov-t) was incorrectly added to the original [License](https://github.com/StayInTarkov/StayInTarkov.Client/commit/a2d111eeac96b4ddb15f5fc5a182330f25ddbe4f#diff-4673a3aba01813b595de187a7a6e9e63a3491d55821606fecd9f13a10c188a1d) with the message `Paulov-t - Original Stay In Tarkov (SIT) Code`. Original Stay in Tarkov (SIT) code means code merged from the [Paulov-t GitHub](https://github.com/paulov-t) and no code developed after the fact. You are therefore accepting LICENSE any original code written up to the point of the move to Stay In Tarkov organisation.
```

Comment:
`LICENSE.md` up to this point was the only lawful license present in the repository, therefore any code up to this point have been covered by this license.